repository/RediStick-Connection-Pool-Tests.package/RsRedisConnectionPoolTest.class/instance/initializeConnectionPool.st

```diff
@@ -0,0 +1,4 @@
+running
+initializeConnectionPool
+	RsRedisConnectionPool initialize.
+	connectionPool := RsRedisConnectionPool default
```

Comment:
In this case, `RsRedisConnectionPool new` would be better. By using this, we can keep the actual (default) connection pool intact even when the test case stops with some unexpected error.

Reply:
done 66f4d7d